Mathlib/AlgebraicGeometry/Gluing.lean

```diff
@@ -390,7 +390,7 @@ instance : Epi 𝒰.fromGlued.val.base := by
   intro x
   obtain ⟨y, h⟩ := 𝒰.covers x
   use (𝒰.gluedCover.ι (𝒰.f x)).1.base y
-  erw [← comp_apply] -- now `erw` after #13170
+  rw [← comp_apply]
```

Comment:
Do these two calls combine? (The other new changes also look good to me.)

Reply:
They do not (note that the other one is `at h`), but I spotted two others that do! Thanks for checking :D

Reply:
(I should not forget to push my changes...)